server/Events/AllEvents.py

```diff
@@ -12,20 +11,54 @@
     'db': 'pye_data'  
 }
 
+# Function to assign label based on event title
+def assign_event_label(title):
+    title_lower = title.lower()
+    
+    # Technology & IT
+    if any(keyword in title_lower for keyword in ['technology', 'it', 'tech', 'software', 'programming', 'development', 'coding', 'ai']):
```

Comment:
Ideally it would be better if the values used in keyword would be pulled from db, but because of short time we can look over this for now, however if you have extra time please start working in that direction.

other than that great work!